### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by samanthabaker**

### CODE REVIEW

**Comment by samanthabaker** [comment]
Code clean up
Moved toolbar styles to toolbar.scss

File: src/components/03-components/09-accordion/accordion.njk [hunk 1]
```diff
@@ -4,35 +4,34 @@
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:4]: Use respective elements in Sam Styles (ie: buttons, accordions, etc)

File: src/components/03-components/09-accordion/accordion.njk [hunk 2]
```diff
@@ -4,35 +4,34 @@
 
   <!-- Use the accurate heading level to maintain the document outline -->
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:5]: Make reusable and use throughout Sam Styles to ensure consistency

Comment (ascreven-gsa) [reply src/components/03-components/09-accordion/accordion.njk:5]: Use variants and make collated true in config file

File: src/stylesheets/components/_accordion.scss [hunk 3]
```diff
@@ -1,251 +1,90 @@
...
-
-.sds-accordion__trigger:focus .sds-accordion__title {
-  //border-color: hsl(216, 94%, 73%);
-}
-.sds-accordion__title-text {
-  display: inline-block;
-  font-weight: 400;
-  div:last-child{
-    font-weight: 600;
+.usa-accordion,
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:1]: Remove accordion code from _sds-theme-custom-styles.scss

File: src/stylesheets/components/_accordion.scss [hunk 4]
```diff
@@ -1,251 +1,90 @@
...
+    @include u-position('relative');
+    background-image: none;
+
+    &:hover {
+      @include u-bg('accent-cool-light')
+    }
+
+    &:focus,
+    &:not([disabled]):focus {
+      outline: none;
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:18]: Use u-outline mixins https://designsystem.digital.gov/utilities/outline/

File: src/stylesheets/components/_accordion.scss [hunk 5]
```diff
@@ -1,251 +1,90 @@
...
-.sds-accordion__trigger:focus .sds-accordion__title {
-  //border-color: hsl(216, 94%, 73%);
-}
-.sds-accordion__title-text {
-  display: inline-block;
-  font-weight: 400;
-  div:last-child{
-    font-weight: 600;
+.usa-accordion,
+.usa-accordion.usa-accordion--bordered {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:2]: use color variables or create new ones throughout for consistency

File: src/stylesheets/components/_toolbar.scss [hunk 6]
```diff
@@ -163,4 +163,194 @@
...
+.sds-accordion__trigger {
+  display: block;
+  font-size: 17px;
+  font-weight: 600;
+  margin: 0;
+  padding: 0;
+  position: relative;
+  text-align: left;
+  width: 100%;
+  outline: none !important;
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:175]: remove important tag

File: src/stylesheets/components/_toolbar.scss [hunk 7]
```diff
@@ -163,4 +163,194 @@
   margin-left: 20px;
 }
 
+.sds-accordion__trigger {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:166]: Put in accordion style file

File: src/stylesheets/components/_toolbar.scss [hunk 8]
```diff
@@ -16,7 +16,7 @@
     text-decoration: none;
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:16]: use color variables or create new ones throughout for consistency

File: src/stylesheets/components/_toolbar.scss [hunk 9]
```diff
@@ -163,4 +163,194 @@
...
+.sds-accordion__title {
+  display: block;
+  pointer-events: none;
+  border: transparent 1px dashed;
+  border-radius: 2px;
+  outline: none;
+  padding: 6px 10px;
+}
+
+.sds-accordion__trigger:focus .sds-accordion__title {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:199]: remove unused code

File: src/stylesheets/components/_toolbar.scss [hunk 10]
```diff
@@ -163,4 +163,194 @@
...
+.sds-accordion__trigger:focus,
+.sds-accordion__trigger:hover {
+  background: #eff6fb;
+}
+
+.sds-accordion .sds-accordion__trigger::-moz-focus-inner {
+  border: 0;
+}
+
+.sds-accordion__title {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:190]: use mixins

File: src/stylesheets/components/_toolbar.scss [hunk 11]
```diff
@@ -163,4 +163,194 @@
...
+  padding: 8px 5px;
+}
+
+/* Accordion -- Basic */
+.sds-accordion.sds-accordion--basic {
+  border-color: transparent;
+  padding-bottom: 0;
+}
+
+.sds-accordion.sds-accordion--basic.focus {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:295]: remove unused code

File: src/stylesheets/components/_toolbar.scss [hunk 12]
```diff
@@ -163,4 +163,194 @@
...
+  border: 0;
+}
+
+.sds-accordion__title {
+  display: block;
+  pointer-events: none;
+  border: transparent 1px dashed;
+  border-radius: 2px;
+  outline: none;
+  padding: 6px 10px;
```
Comment (davereed) [inline src/stylesheets/components/_toolbar.scss:196]: Can we update these to units?

I used this for the matching filter panel (4px and 8px):
padding: units('05') units(1);

Comment (samanthabaker) [reply src/stylesheets/components/_toolbar.scss:196]: toolbar component

File: src/components/03-components/09-accordion/accordion.njk [hunk 13]
```diff
@@ -1,38 +1,35 @@
+{#
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:1]: Remove commented out code

File: src/components/03-components/09-accordion/accordion.njk [hunk 14]
```diff
@@ -43,67 +40,60 @@
...
+    <p>No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or
+      public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due
+      process of law; nor shall private property be taken for public use, without just compensation.</p>
 
   </div>
 
 </div>
+#}
 
-<h6>Bordered</h6>
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:72]: Make variant so it's reusable

File: src/components/03-components/09-accordion/accordion.njk [hunk 15]
```diff
@@ -113,29 +103,96 @@
...
+    <p>No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or
+      public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due
+      process of law; nor shall private property be taken for public use, without just compensation.</p>
+
+  </div>
+
+</div>
+
+<h6>Multiselectable</h6>
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:132]: Make reusable variant

File: src/components/03-components/09-accordion/accordion.njk [hunk 16]
```diff
@@ -1,38 +1,35 @@
+{#
 <h6>Borderless</h6>
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:2]: render where used (landing page, footer, banner, alerts, nav, toolbar)

File: src/stylesheets/components/_accordion.scss [hunk 17]
```diff
@@ -1,251 +1,86 @@
...
-/* stylelint-enable */
-
-// usa accordion styles button border
-
-.usa-accordion__heading.sds-accordion__heading {
-  margin-top: 0 !important;
-
-  &:first-child {
-    border-top: solid 1px #c9c9c9;
+.usa-accordion.usa-accordion--bordered{
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:67]: repetitive of section above

File: src/stylesheets/components/_accordion.scss [hunk 18]
```diff
@@ -1,251 +1,86 @@
...
-.sds-accordion__trigger:focus .sds-accordion__title {
-  //border-color: hsl(216, 94%, 73%);
-}
-.sds-accordion__title-text {
-  display: inline-block;
-  font-weight: 400;
-  div:last-child{
-    font-weight: 600;
+.usa-accordion,
+.usa-accordion.usa-accordion--bordered {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:2]:
```
.usa-accordion {
&--bordered {

```

File: src/stylesheets/components/_accordion.scss [hunk 19]
```diff
@@ -1,251 +1,86 @@
...
+  @include u-padding-x(1);
+  .usa-accordion__button {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:4]: center buttons with text

File: src/stylesheets/components/_accordion.scss [hunk 20]
```diff
@@ -1,251 +1,86 @@
...
+      height: 25px;
+      background: linear-gradient(#0a3466,#0a3466);
+      background-position: center;
+      background-size: 54% 1px;
+      background-repeat: no-repeat;
+    }
+
+    &[aria-expanded=true] {
+      background-image: none;
+      color: #0a3466;
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:40]: use taken

File: src/stylesheets/components/_accordion.scss [hunk 21]
```diff
@@ -1,251 +1,86 @@
...
+
+    &:focus,
+    &:not([disabled]):focus {
+      @include u-outline('secondary-lighter');
+      @include u-outline('1px');
+      outline-style: dashed;
+      outline-offset: 0;
+    }
+
+    &::after {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:24]: Why isn't this an icon?

File: src/components/03-components/09-accordion/accordion.njk (hunk 13 above)
Comment (samanthabaker) [reply src/components/03-components/09-accordion/accordion.njk:1]: I was specifically asked to keep that in and commented out

File: src/stylesheets/components/_accordion.scss (hunk 17 above)
Comment (samanthabaker) [reply src/stylesheets/components/_accordion.scss:67]: calling different classes

File: src/stylesheets/components/_accordion.scss (hunk 19 above)
Comment (samanthabaker) [reply src/stylesheets/components/_accordion.scss:4]: accordion button wouldn't have text

File: src/stylesheets/components/_accordion.scss (hunk 21 above)
Comment (samanthabaker) [reply src/stylesheets/components/_accordion.scss:24]: it's USWDS styling

File: src/stylesheets/components/_toolbar.scss [hunk 22]
```diff
@@ -163,4 +163,194 @@
   margin-left: 20px;
 }
 
+.sds-accordion__trigger {
+  display: block;
+  font-size: 17px;
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:168]: same as above with colors and sizing variables

File: src/stylesheets/components/_toolbar.scss (hunk 7 above)
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:166]: move to _accordion.scss

File: src/stylesheets/components/_accordion.scss [hunk 23]
```diff
@@ -1,251 +1,87 @@
...
-    @include u-padding-right(6);
-    color: color('ink');
-    font-size: 1.25rem;
-  }
-
-  button.usa-accordion__block[aria-expanded=true] {
-    @include u-bg('white');
-    color: color('ink');
-    @include u-border-bottom(0);
+  .usa-accordion__button {
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_accordion.scss:72]: button isn't centered in heading

File: src/stylesheets/components/_toolbar.scss (hunk 10 above)
Comment (ascreven-gsa) [inline src/stylesheets/components/_toolbar.scss:190]: remove classes that no longer exist

File: src/components/03-components/09-accordion/accordion.njk [hunk 24]
```diff
@@ -1,142 +1,101 @@
...
 
-<div class="sds-accordion usa-accordion">
-
-  <!-- Use the accurate heading level to maintain the document outline -->
-  <h2 class="usa-accordion__heading sds-accordion__heading">
-    <button class="usa-accordion__button usa-accordion__block"
-      aria-expanded="true"
-      aria-controls="b-a1">
-      First Amendment
+  <h2 class="usa-accordion__heading">
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:56]: turn into a for loop for reuse in other components

File: src/components/03-components/09-accordion/accordion.njk [hunk 25]
```diff
@@ -1,142 +1,101 @@
...
-    <p>No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due process of law; nor shall private property be taken for public use, without just compensation.</p>
-
+    <p>{{header}}</p>
   </div>
-
 </div>
+#}
 
-<h6>Bordered</h6>
+<div class="usa-accordion usa-accordion--bordered" {% if multiselect %} aria-multiselectable="true"{% endif %}>
```
Comment (ascreven-gsa) [inline src/components/03-components/09-accordion/accordion.njk:54]: add in components using accordions for consistency/to ensure proper configuration

**Pull request merged**